modules/theora/SCsub

```diff
@@ -22,49 +22,49 @@ if env["builtin_libtheora"]:
         "decode.c",
         "dequant.c",
         # "encapiwrapper.c",
-        # "encfrag.c",
-        # "encinfo.c",
-        # "encode.c",
+        "encfrag.c",
+        "encinfo.c",
+        "encode.c",
```

Comment:
Is there any use case for having the theora encoder at runtime in export templates?

If we need it only for the editor, we should probably gate compiling these files (and adding the `theoraenc` and `vorbisenc` dependencies in `linuxbsd/detect.py`) on `env.editor_build`.

This helps keep export templates compat.